core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/RawlsBillingProjectComponent.scala

```diff
@@ -144,6 +151,25 @@ trait RawlsBillingProjectComponent {
     ) <> ((RawlsBillingProjectRecord.apply _).tupled, RawlsBillingProjectRecord.unapply)
   }
 
+  class WorkspaceManagerResourceMonitorTable(tag: Tag) extends Table[WorkspaceManagerResourceMonitor](tag, "WORKSPACE_MANAGER_RESOURCE_MONITOR") {
+    def jobControlId = column[UUID]("JOB_CONTROL_ID", O.PrimaryKey)
+
+    def jobType = column[String]("JOB_TYPE")
+
+    def workspaceId = column[Option[UUID]]("WORKSPACE_ID")
+
+    def billingProjectId = column[Option[String]]("BILLING_PROJECT_ID")
+
+    def createdTime = column[Instant]("CREATED_TIME")
+
+    def * = (jobControlId,
+      jobType,
+      workspaceId,
+      billingProjectId,
+      createdTime
+    ) <> ((WorkspaceManagerResourceMonitor.apply _).tupled, WorkspaceManagerResourceMonitor.unapply)
```

Comment:
@marctalbott do we need to add anything in the model for the foreign keys/referenced tables (I'm not that familiar with slick)?

Reply:
Yes: here's the [documentation for foreign keys](https://scala-slick.org/doc/3.0.0/schemas.html#constraints)

Reply:
Hmm, looks like Slick does provide [additional FK support](https://scala-slick.org/doc/3.0.0/schemas.html#constraints), but we don't use it elsewhere in Rawls afaik and it doesn't appear to provide many benefits in addition to the FK constraint defined in the actual DB. I think this is fine as it is